elytron/src/test/resources/org/wildfly/extension/elytron/credential-security-factories.xml

```diff
@@ -16,6 +16,7 @@
                                    server="false"
                                    obtain-kerberos-ticket="true"
                                    debug="true"
+                                   mechanisms="KRB5 KRB5LEGACY"
```

Comment:
+1 for option to set unusual/custom GSS mechanism. 

But could not that be achieved by possibility to pass pure OID into mechanisms? E.g. `mechanisms="KRB5 SPNEGO 1.2.3.4.5.6"`

If not, please:
*  add allowed values for `mechanisms` attribute
*  add into description, that `mechanism-oids` are like addition to `mechanisms` attribute - that first extends second. Something like

`elytron.kerberos-security-factory.mechanism-oids=Possibility to specify pure OIDs in addition to specified mechanism names in mechanisms attribute`